### DISCUSSION TITLE
Bib file does not include volume titles

### DISCUSSION THREAD

**Discussion by moiken-hinrichs** [retorquere/zotero-better-bibtex#2227] [Q&A]
Hello,
I'm working with RMarkdown/Bookdown and Zotero as my library, which generally is working quite nice. I just noted, that all the volume titles are missing in the bibliography after rendering the document. First I thought it's the .csl which is at fault (using taylor-and-francis-harvard-v) but when I checked the .bib file, I saw that the titles were not included.

An exampe:
```
@inbook{pelegrin2000,
	title = {Les techniques de débitage laminaire au Tardiglaciaire: critères de diagnose et quelques réflexions},
	author = {Pelegrin, Jacques},
	editor = {Valentin, Boris and Bodu, Pierre and Christensen, Marianne},
	year = {2000},
	date = {2000},
	pages = {73{\textendash}86},
	series = {Mémoires du Musée de Préhistoire d'lle de France},
	address = {Nemours}
}
```
Thats how the file looks for all the @inbook, clearly missing the title after the editior. Is there a way I can change that, so I don't have to manualy add the titles after finishing the writing process?

**Comment by retorquere** [comment]
I would have to change it. Please post a support log ID on #2228 by right-clicking the zotero entry and selecting "send support log" from the BBT submenu.

**Reply by moiken-hinrichs** [reply]
Hopefully I managed to support the log in the right way. The ID is: P32R5Z9G-refs-euc

**Reply by retorquere** [reply]
That is the correct ID, but I'd prefer to continue the dialogue at #2228
